docs/crio.conf.5.md

```diff
@@ -287,6 +287,9 @@ The `crio.metrics` table containers settings pertaining to the Prometheus based
 **enable_metrics**=false
   Globally enable or disable metrics support.
 
+**enable_custom_shm_size**=false
+If set to true, enable users to set a custom shm size instead of using the default value of 64M. The shm size can be set through K8S annotation with the key "io.kubernetes.cri-o.ShmSize", and the value representing the size in human readable format. For example: "io.kubernetes.cri-o.ShmSize: 128Mi"
+
```

Comment:
The last couple of sentences aren't in https://github.com/cri-o/cri-o/pull/3970/files#diff-1449dc4b21cb9690f2ab9ed09cff03bdR170, could/should it be?

Reply:
yes, it's been fixed